File: 2020/2-22-Scrapy-Puppeteer-Js/js_server.go
/*
@File    :   js_server.go
@Time    :   2020/02/22 11:32:32
@Author  :   play4fun
@Desc    :   d
*/

package main

import (
	"fmt"
	"html/template"
	"net/http"
)

func main() {
	fmt.Println("Open http://127.0.0.1:8888/")
	tmpl := template.Must(template.ParseFiles("index.html"))

	data := struct {
		Title string
		// Items []string
	}{
		Title: "Js动态页面演示",
		// Items: []string{
		// 	"My pictures",
		// 	"My dialog",
		// },
	}

	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
		tmpl.Execute(w, data)
	})

	http.ListenAndServe("0.0.0.0:8888", nil)
}

File: 2020/2-22-Scrapy-Puppeteer-Js/js_server.py
# -*- encoding: utf-8 -*-
'''
@File    :   js_server.py
@Time    :   2020/02/23 09:53:30
@Author  :   play4fun
@Desc    :   https://www.w3cschool.cn/flask/flask_templates.html
'''

from flask import Flask, render_template
app = Flask(__name__)


@app.route('/', methods=["GET"])
def index():
    Title = "Js动态页面演示"
    return render_template('index.html', Title=Title)


if __name__ == '__main__':
    app.run(host='0.0.0.0',port=8888,debug=True)
